addons/sys_rack/fnc_addRackOnReturn.sqf

```diff
@@ -16,14 +16,18 @@
  */
 #include "script_component.hpp"
 
-params ["_vehicle","_rackId"];
+params ["_vehicle", "_rackId"];
 
-private _rack = _rackId createVehicle [0,0,0];
+private _rack = _rackId createVehicle [-1000,-1000,-1000];
```

Comment:
That's because AGL and ATL are not the same, but if both use AGL it should work just fine. Either way this should sue the alternative syntax due to `CAN_COLLIDE`.